rts/Game/UI/MiniMap.cpp

```diff
@@ -1675,7 +1682,7 @@ void CMiniMap::DrawButtons()
 void CMiniMap::DrawNotes()
 {
 	RECOIL_DETAILED_TRACY_ZONE;
-	if (notes.empty()) {
+	if (notes.empty() || !drawNotes) {
```

Comment:
This function also manages pings' lifetime (it's the one that removes them) so they will pile up forever. Consider adding `if (!drawNotes)` at the top of `CMiniMap::AddNotification` and `notes.clear()` in `CMiniMap::ConfigCommand`.